src/sortStudents.ts

```diff
@@ -1,16 +1,54 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
-
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+const getAverage = (marks:number[]): number => {
+  const averageMark:number = marks
+    .reduce((prevNum: number, nextNum: number): number => prevNum + nextNum, 0)
+    / marks.length;
+
+  return averageMark;
+};
```

Comment:
You can return immideatly
```suggestion
const getAverageGrades = (marks:number[]): number => {
  return marks
    .reduce((prevNum, nextNum): number => prevNum + nextNum, 0)
    / marks.length;

  return averageMark;
};
```